Source/PeleLMeX_BCfill.H

```diff
@@ -117,6 +116,79 @@ struct PeleLMCCFillExtDirState
   }
 };
 
+struct PeleLMCCFillExtDirAux
+{
+
+  ProbParm const* lprobparm;
+  pele::physics::PMF::PmfData::DataContainer const* lpmfdata;
+  const int m_nAux;
+
+  AMREX_GPU_HOST
+  constexpr PeleLMCCFillExtDirAux(
+    ProbParm const* a_prob_parm,
+    pele::physics::PMF::PmfData::DataContainer const* a_pmf_data,
+    int a_nAux)
+    : lprobparm(a_prob_parm), lpmfdata(a_pmf_data), m_nAux(a_nAux)
+  {
+  }
+
+  AMREX_GPU_DEVICE
+  void operator()(
+    const amrex::IntVect& iv,
+    amrex::Array4<amrex::Real> const& aux,
+    const int /*dcomp*/,
+    const int /*numcomp*/,
+    amrex::GeometryData const& geom,
+    const amrex::Real time,
+    const amrex::BCRec* bcr,
+    const int /*bcomp*/,
+    const int /*orig_comp*/) const
+  {
+
+    // Get geometry data
+    const int* domlo = geom.Domain().loVect();
+    const int* domhi = geom.Domain().hiVect();
+    const amrex::Real* prob_lo = geom.ProbLo();
+    const amrex::Real* dx = geom.CellSize();
+    const amrex::Real x[AMREX_SPACEDIM] = {AMREX_D_DECL(
+      prob_lo[0] + (iv[0] + 0.5) * dx[0], prob_lo[1] + (iv[1] + 0.5) * dx[1],
+      prob_lo[2] + (iv[2] + 0.5) * dx[2])};
+
+    amrex::CellData<amrex::Real> aux_ext(aux.ptr(iv), aux.nstride, m_nAux);
```

Comment:
How about `auto aux_ext = aux.cellData(AMREX_D_DECL(iv[0], iv[1], iv[2]));` instead here?

Reply:
I've added this, although you don't need the AMREX_D_DECL.

Reply:
Ah yeah, I figured there was an overload in 2D that just used i,j but because it's indexing into an Array4 it makes sense that it expects i,j,k even when compiled in 2D.

Adding an IntVect version of cellData would be a simple PR into AMReX if you're interested in doing that.